source/features/search-in-current-path.tsx

```diff
@@ -0,0 +1,37 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import GitHubURL from '../github-helpers/github-url';
+
+let search = '';
+
+function init(): void {
+	const searchInput = select<HTMLInputElement>('[data-hotkey="s,/"]')!;
+
+	if (searchInput.value === search) {
+		const {route, filePath} = new GitHubURL(location.href);
+		
+		if (route === 'tree') {
+			searchInput.value = `path:${filePath} `;
+		} else if (filePath.includes('/')) { // Exclude files at the root
+			searchInput.value = `path:${filePath.slice(0, filePath.lastIndexOf('/'))} `;
```

Comment:
I haven't tested my last commit, but this should make it work correctly on files on the root of branches with slashes. `GitHubURL` was specifically built to consider this.

Test URL: https://github.com/sindresorhus/refined-github/blob/slash/dot/license

Reply:
Tested, it works well 🎉